ckanext/canada/schemas/harvest.yaml

```diff
@@ -0,0 +1,112 @@
+scheming_version: 2
+dataset_type: harvest
+
+
+form_languages:
+- en
+- fr
+# machine translated metadata:
+alternate_languages:
+  en:
+    - en-t-fr
+  fr:
+    - fr-t-en
+
+
+sidebar_show_fields:
+- url
+- source_type
+- frequency
+
+
+#
+#   DATASET FIELDS
+#
+dataset_fields:
+
+
+- field_name: id
+  create_validators: canada_validate_generate_uuid
+      unicode name_validator package_id_does_not_exist portal_sync_id portal_sync_limit
```

Comment:
If you want to prevent more than one from being created you could use a validator that always returns the same id

Reply:
I initially tried doing that, but then got some SQL errors because the Harvest plugin does some crazy table reference stuff during package creation